test/integration/spec/remotetracks.js

```diff
@@ -108,7 +108,7 @@ describe('RemoteParticipant', function() {
 
     // eslint-disable-next-line no-warning-comments
     // TODO: enable these tests when track_priority MSP is available in prod
-    if (defaults.topology !== 'peer-to-peer' && defaults.environment === 'dev') {
+    if (defaults.topology !== 'peer-to-peer' && defaults.environment === 'stage') {
```

Comment:
@makarandp0 This condition should be applied at the top-level `describe`:
```js
(defaults.topology === 'peer-to-peer' || defaults.environment === 'prod'
  ? describe.skip
  : describe)('#setPriority', () => {...});
```
Also, please change `describe('RemoteParticipant', ...)` to `describe('RemoteVideoTrack', ...)`.